### PR TITLE
turn on prettier for .md files (other than docs)

### PR DESCRIPTION
**Pull request by johannkm**
per title

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #18342** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/18342?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/18342?utm_source=stack-comment).

File: examples/README.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ This directory contains some example projects that showcase how Dagster is used
 
 Some of these are actively maintained as new features are added to Dagster, while those that are labeled with the `UNMAINTAINED` tag in their README are valid projects but may not be actively maintained.
 
-If you'd like to see a larger scale Dagster project, check out our own analytics project: [Dagster Open Platform](https://github.com/dagster-io/dagster-open-platform)
```
Comment (johannkm) [inline examples/README.md:7]: dunno the difference here... carriage return?

**Review by alangenfeld — APPROVED** [review]
`mdx` next?

**Comment by alangenfeld** [comment]
why avoid `docs` md files?

**Comment by johannkm** [comment]
Given the separate suite of docs formatting with make full-format, it
seemed best to keep this separate. Maybe an alternative would be for make
docs-full-format to call the top level make prettier

**Pull request merged**